akka-docs/rst/general/addressing.rst

```diff
@@ -143,6 +139,18 @@ systems or JVMs. This means that the logical path (supervision hierarchy) and
 the physical path (actor deployment) of an actor may diverge if one of its
 ancestors is remotely supervised.
 
+
+Actor path alias or symbolic link?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+As in some real file-systems you might think of a "path alias" or "symbolic link”for an actor,
```

Comment:
Ah sorry I used inconsistent ones. Thanks for finding it out. I fixed it in the latest commit.